src/acceptance/app/app_suite_test.go

```diff
@@ -208,3 +240,102 @@ func generateDynamicScaleInPolicy(instanceMin, instanceMax int, threshold int64)
 
 	return string(bytes)
 }
+
+func generateDynamicAndSpecificDateSchedulePolicy(instanceMin, instanceMax int, threshold int64,
```

Comment:
Why are we naming the method generateDynamicAndSpecificDateSchedulePolicy and not generateSpecificDateSchedulePolicy